po/fr.po

```diff
@@ -78466,7 +78487,7 @@ msgid "specify a key/value option"
 msgstr ""
 
 msgid "specify a new shell, defaults to '/bin/bash -l -c'"
-msgstr "Spécifier un nouveaux shell, '/bin/bash -l -c' par défaut"
+msgstr "Spécifier un nouveaux shell, « /bin/bash -l -c » par défaut"
```

Comment:
> I’d like to address them all before asking for merge.

these are massive -- 26k translatable strings, from various technical domains.
Good translations take a lot of effort.
So we're definitely happy to merge intermediate results.

Most are from share/completions/.
We could maybe classify those files to separate barely-used tools from others (based on Debian popcon or similar install counts), so translators can focus on the most meaningful translations.